php/class-responsive-breakpoints.php

```diff
@@ -0,0 +1,336 @@
+<?php
+/**
+ * Responsive breakpoints.
+ *
+ * @package Cloudinary
+ */
+
+namespace Cloudinary;
+
+use Cloudinary\Component\Assets;
+use Cloudinary\Component\Setup;
+use Cloudinary\Connect\Api;
+
+/**
+ * Class Responsive_Breakpoints
+ *
+ * @package Cloudinary
+ */
+class Responsive_Breakpoints implements Setup, Assets {
```

Comment:
@DavidCramer 

It seems that this class is addressing both Responsive Breakpoints and Lazy Loading concepts.
What do you think of separate these concerns?